Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -1614,6 +1614,17 @@ def get_indicator_conditions(client: Client, args: Dict[str, Any]) -> CommandRes
     )
 
 
+def validation_baseurl(baseurl: str) -> None:
+    error_message = ''
+    for suffix in (('/v3', '/v3/'), ('/api', '/api/'), ('/hx', '/hx/')):
+        if baseurl.endswith(suffix):
```

Comment:
no need for the for loop, unindent it all and replace with 

```suggestion
    if baseurl.endswith(('/v3', '/v3/', '/api', '/api/', '/hx', '/hx/')):
```